api/proposal.go

```diff
@@ -30,10 +30,66 @@ import (
 	"k8s.io/enhancements/pkg/yaml"
 )
 
-var ValidStages = []string{
-	"alpha",
-	"beta",
-	"stable",
+type Stage string
+
+const (
+	AlphaStage  Stage = "alpha"
+	BetaStage   Stage = "beta"
+	StableStage Stage = "stable"
+)
+
+var ValidStages = []Stage{
+	AlphaStage,
+	BetaStage,
+	StableStage,
+}
+
+func (s Stage) String() string {
+	return string(s)
+}
```

Comment:
We can just use `string(<str>)` on the caller sides. Then remove this.

(same for Status)

Reply:
done